types/node-telegram-bot-api/index.d.ts

```diff
@@ -1388,6 +1405,26 @@ declare class TelegramBot extends EventEmitter {
             'webhook_error' |
             'error'
     ): number;
+
+  setChatPermissions(
+    chatId: number | string,
+    chatPermissions: TelegramBot.ChatPermissions
+  ): Promise<boolean>;
+
+  sendDice(
+    chatId: number | string,
+    options?: TelegramBot.SendDiceOptions
+  ): Promise<TelegramBot.Message>;
+
+  setChatAdministratorCustomTitle(
+    chatId: number | string,
+    userId: string,
+    customTitle: string
+  ): Promise<boolean>;
+
+  getMyCommands(): Promise<TelegramBot.BotCommand>;
```

Comment:
Is it `TelegramBot.BotCommand` or `Array<TelegramBot.BotCommand>`? The [docs](https://core.telegram.org/bots/api#getmycommands) says:

> Returns Array of BotCommand on success.

Reply:
Yeah, my bad, i'll fix it.